crates/hyperswitch_connectors/src/connectors/fiserv/transformers.rs

```diff
@@ -1,15 +1,24 @@
+use common_enums::enums;
 use common_utils::{ext_traits::ValueExt, pii};
 use error_stack::ResultExt;
+use hyperswitch_domain_models::{
+    payment_method_data::PaymentMethodData,
+    router_data::{ConnectorAuthType, RouterData},
+    router_flow_types::refunds::{Execute, RSync},
+    router_request_types::ResponseId,
+    router_response_types::{PaymentsResponseData, RefundsResponseData},
+    types,
+};
+use hyperswitch_interfaces::{api, errors};
+use masking::Secret;
 use serde::{Deserialize, Serialize};
 
 use crate::{
-    connector::utils::{
+    types::{RefundsResponseRouterData, ResponseRouterData},
+    utils::{
         self, CardData as CardDataUtil, PaymentsCancelRequestData, PaymentsSyncRequestData,
-        RouterData,
+        RouterData as RouterDataUtil,
```

Comment:
You can import traits as `_`, if you'd prefer:

```suggestion
        RouterData as _,
```

Reply:
Done